perfkitbenchmarker/openstack/os_virtual_machine.py

```diff
@@ -34,6 +35,11 @@
 flags.DEFINE_integer('openstack_volume_size', 20,
                      'Size of the volume (GB)')
 
+flags.DEFINE_enum('openstack_scheduler_policy', NONE,
```

Comment:
Sorry, I saw your commit history. If the point of this change was because None is incompatible with DEFINE_enum, then how about changing the casing to match the 'affinity' and 'anti-affinity' options?